.github/workflows/ci.yaml

```diff
@@ -6,11 +6,13 @@ on:
       - main
       - v[0-9]+
       - v[0-9]+.[0-9]+
+      - cryostat-v[0-9]+.[0-9]+
   pull_request:
     branches:
       - main
       - v[0-9]+
       - v[0-9]+.[0-9]+
+      - cryostat-v[0-9]+.[0-9]+
```

Comment:
Just noticing now but does the `.` here need to be escaped so that it matches a literal `.` character and not an "any-character"? I think this might match ex. `cryostat-v2a2`. I doubt we'll ever have branch names of that form, but still.

Reply:
It doesn't seem like `.` has special meaning in GitHub's syntax: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet